src/Bicep.LangServer/Handlers/BicepDeployCommandHandler.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.ResourceManager;
+using Bicep.Core.Configuration;
+using Bicep.Core.Registry.Auth;
+using Bicep.LanguageServer.Deploy;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    public class BicepDeployCommandHandler : ExecuteTypedResponseCommandHandlerBase<string, string, string, string, string, string, string>
```

Comment:
Would be much easier to understand if you implemented a record for parameters, see for instance BicepDeploymentGraphHandler.